src/components/process-timestamp-view.js

```diff
@@ -135,7 +127,6 @@ const ProcessTimestampView = () => {
             <Grid item>
                 <TableCore
                     taskData={taskData}
-                    gridCapaJobLauncherUrl={gridcapaJobLauncherUrl}
```

Comment:
same remark (and for all modifications where you remove gridCapaJobLauncherUrl
PS: if you keep this field, it would be better to remove gridCapa from its name